abci/example/counter/counter.go

```diff
@@ -84,3 +84,5 @@ func (app *Application) Query(reqQuery types.RequestQuery) types.ResponseQuery {
 		return types.ResponseQuery{Log: fmt.Sprintf("Invalid query path. Expected hash or tx, got %v", reqQuery.Path)}
 	}
 }
+
+// XXX: add voteext
```

Comment:
What does this mean and is it required?

Reply:
For VoteExtension impl in the following PRs, I will just remove the comments